README.md

```diff
@@ -18,7 +18,7 @@ radio systems.
 ### From Source
 
 ```
-git clone https://github.com/gnuradio --branch dev-4.0
+git clone https://github.com/gnuradio.git --branch dev-4.0
```

Comment:
actually, shouldn't this be github.com/gnuradio/gnuradio
(I think the .git is optional)

Reply:
thank you for catching the original typo here!

Reply:
Fixed